src/content/docs/api/protocols/http.mdx

```diff
@@ -0,0 +1,366 @@
+---
+title: HTTP API
+description: Documentation for the HTTP API
+---
+
+You can control most of the playback functions of Ontime over HTTP protocol. \
+This could be handy for integration with third party software, [including vMix](/additional-notes/control-from-vmix).
+
+:::note[Notes on requests]
+All HTTP requests are **GET** requests. \
+Please mind the correct configuration for IP and Port will depend on your setup.
+
+Any request which successfully changes the Ontime playback will have a status code in the 200 range.
+:::
+
+
+:::tip[Show me the code]
+This documentation is generally simplified. In truth, the HTTP API is more complex. \
+As Ontime uses the HTTP API to manage its data, this changes often.
+
+You can follow the code for a more up-to-date reference:
+[Get data from Ontime](https://github.com/cpvalente/ontime/tree/master/apps/server/src/api-data)
+[Control Ontime](https://github.com/cpvalente/ontime/tree/master/apps/server/src/api-integration)
+:::
+
+
+## State
+The following endpoints allow querying Ontime for its current state
+
+### Get Ontime version
+**Request** 
+```bash
+<localhost:4001>/api/version
+```
+
+**Response**
+```ts
+{"payload":"3.0.0"}
+```
+
+### Get Ontime runtime state
+**Request** 
+```bash
+<localhost:4001>/api/poll
+```
+
+**Response** \
+The response of a poll request is a [runtime data object](/api/data/runtime-data)
+```ts
+{"payload": <runtime-data>}
+```
+
+### Data from Ontime
+
+With the HTTP API, in addition to controlling the application, you can also retrieve its data. \
+There is no expected payload for these GET requests
+
+| Address                | Description                                                  |
+| :--------------------- | :----------------------------------------------------------- |
+| `/data/custom-fields`  | Get registered [custom fields](/features/custom-fields)      |
+| `/data/db`             | Get currently loaded project file                            |
+| `/data/http`           | Get HTTP integration settings                                |
+| `/data/osc`            | Get OSC integration settings                                 |
+| `/data/project`        | Get current project data                                     |
+| `/data/rundown`        | Get current rundown                                          |
+| `/data/settings`       | Get current application settings                             |
+| `/data/url-presets`    | Get currently defined [URL Presets](/features/url-presets)   |
+| `/data/view-settings`  | Get currentluy defined settings for views                    |
+
+
+## Change event
+```bash
+<localhost:4001>/api/change/<event-id>/<property>/<value>
```

Comment:
it is also possible to send multiple
`/api/change/92cfdd?title=test&cue=test`

Reply:
hm, I wonder if the mix between query parameters `?param=value `and path parameters `/query/value` is confusing,
even if we support both, should we simplify documentation for only one type?

so in this example
`<localhost:4001>/api/change/<event-id>?<property>=<value>`

what do you think?
I am also not sure we should support both...

Reply:
I think we should recommend the use of the query parameters  
but us trying to prevent the filed/value notation might end up requiring more logic than not....

Reply:
Makes sense. 

I will cleanup. 

In the bug report for the changes endpoint we should just make sure we check the payload before setting 
No need to make it work for url parameters